domain/cosmos/tx/tx.go

```diff
@@ -119,7 +119,7 @@ func SimulateMsgs(
 	txFactory = txFactory.WithAccountNumber(account.AccountNumber)
 	txFactory = txFactory.WithSequence(account.Sequence)
 	txFactory = txFactory.WithChainID(chainID)
-	txFactory = txFactory.WithGasAdjustment(1.05)
+	txFactory = txFactory.WithGasAdjustment(1.15)
```

Comment:
Shall we define a const for this so that it is not like being "buried" somewhere in the code?